py/desitarget/mock/io.py

```diff
@@ -300,6 +300,71 @@ def read_wd100pc(mock_dir, target_type, mock_name=None):
 
     mock_name: :class:`str`
         Optional name of the mock file.
+        default: 'mock_100pc.fits'
+
+    brickname_list:
+        Optional list of specific bricknames to read.
+
+    Returns:
+    -------
+    Dictionary with the following entries.
+
+        'RA': :class: `numpy.ndarray`
+            RA positions for the objects in the mock.
+        'DEC': :class: `numpy.ndarray`
+            DEC positions for the objects in the mock.
+        'Z': :class: `numpy.ndarray`
+            Heliocentric radial velocity divided by the speed of light.
+        'magg': :class: `numpy.ndarray`
+            Apparent magnitudes in Gaia G band
+    """
+    desitarget.io.check_fitsio_version()
+    C_LIGHT = 299792.458
+
+    if mock_name is None:
+        filename  = os.path.join(mock_dir,'mock_100pc.fits')
+    else:
+        filename = os.path.join(mock_dir, mock_name+'.fits')
```

Comment:
details: docstring "name of the mock file" implies that the input should include the extension too, but this adds '.fits'.  I suggest to either not add '.fits' or to check if it is already there before adding.